ichnaea/log.py

```diff
@@ -22,9 +22,11 @@ def configure_logging():
 
     if local_dev_env:
         level = "DEBUG"
+        celery_level = "INFO"
         handlers = ["console"]
     else:
         level = "INFO"
+        celery_level = "WARNING"
```

Comment:
@jwhitlock suggested setting this to WARNING in stage/prod and INFO in local dev environment. I figured I'd follow that advice. If we ever need to change it, we can make it a setting or something.

Reply:
When testing the prod-like settings, the scheduler still says it is logging as ``INFO``:

```
scheduler_1  | __    -    ... __   -        _
scheduler_1  | LocalTime -> 2019-10-02 16:06:15
scheduler_1  | Configuration ->
scheduler_1  |     . broker -> redis://redis:6379/0
scheduler_1  |     . loader -> celery.loaders.app.AppLoader
scheduler_1  |     . scheduler -> celery.beat.PersistentScheduler
scheduler_1  |     . db -> /var/run/location/celerybeat-schedule
scheduler_1  |     . logfile -> [stderr]@%INFO
scheduler_1  |     . maxinterval -> 5.00 minutes (300s)
```

I believe this is because of ``--loglevel INFO`` in ``scripts/run_scheduler.sh``. However, the output is consistent with ``WARNING`` level output. This is confusing, may bite us later.

I overrode ``celery_level = "INFO"`` in the prod-like mode, and I get the many expected lines of the ugly mozlog format, so this ``ichnaea/log.py`` code is setting the log level, not the command line.